File: P04 - Inherirance/Python/Kereta.py
from Kendaraan import Kendaraan

class Kereta(Kendaraan):
    def __init__(self) -> None:
        super().__init__()
        self.tenaga_penarik: str = "Kuda"
        self.jml_penarik: int = 2
        self._kecepatan_maks = 50
        self._kapasistas_penumpang = 8
        
        

File: P04 - Inherirance/Java/Kereta.java
package Praktikum.P04.P04c_Latihan_Inheritance_Java_227008_Elmosius_Suli;
// File : Kereta.java
// Nama : Elmosius Suli
// NRP  : 2272008
// Kelas : B
// Ket :
public class Kereta extends Kendaraan {
    public String tenagaPenarik;
    public int jmlPenarik;

    // constructor
    public Kereta(){
        this.tenagaPenarik = "Kuda";
        this.jmlPenarik = 2;
        super.kecepatanMaks = 50;
        super.kapasitasPenumpang = 8;
    }
}
